src/khoj/interface/web/chat.html

```diff
@@ -1839,6 +1840,191 @@
+        }
+
+        function removeFileFilterFromDB(filename) {
+            var conversation_id = document.getElementById("chat-body").dataset.conversationId;
+            console.log(typeof conversation_id)
+            if (!conversation_id) {
+                console.error("Conversation ID not found on chat-body element.");
+                return;
+            }
+
+            return fetch(`/api/chat/conversation/file-filters`, {
+                method: 'DELETE',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({ filename, conversation_id })  // Pass the filename directly
+            })
+            .then(response => {
+                if (!response.ok) {
+                    throw new Error(`HTTP error! status: ${response.status}`);
+                }
+                return response.json();
+            })
+            .then(data => {
+                console.log("Response from server:", data);
+                return data;
+            })
+            .catch(error => {
+                console.error("Error:", error);
+                throw error;
+            });
+        }
+
+        function getFileFiltersFromDB() {
+            // Get the conversation_id from the data attribute
+            var conversation_id = document.getElementById("chat-body").dataset.conversationId;
+
+            // Make sure conversation_id is not undefined or null
+            if (!conversation_id) {
+                console.error("No conversation ID found on chat-body element.");
+                return Promise.reject("No conversation ID found on chat-body element.");
+            }
+
+            // Perform the fetch request
+            return fetch(`/api/chat/conversation/file-filters/${conversation_id}`, {
+                method: 'GET',
+                headers: {
+                    'Content-Type': 'application/json'
+                }
+            })
+            .then(function(response) {
+                console.log("Response status:", response.status); // Log the response status
+
+                if (!response.ok) {
+                    throw new Error(`HTTP error! status: ${response.status}`);
+                }
+
+                return response.json();
+            })
+            .then(function(data) {
+                console.log("Response from server:", data);
+                return data;
+            })
+            .catch(function(error) {
+                console.error("Error:", error);
+                throw error; // Rethrow the error to be handled elsewhere if needed
+            });
+        }
+
+
+        function loadFileFiltersFromDB(){
+            getFileFiltersFromDB()
+            .then(filters => {
+                var selectedFiles = document.getElementsByClassName("selected-files")[0];
+                selectedFiles.innerHTML = "";
+                for (var filter of filters) {
+                    var text = '✅ ' + filter;
```

Comment:
Rather than a checkmark, what if we gave it some neat styling via a custom css class? Something like this, like a stronger border and a glow effect with the yellow-ish color could work.

<img width="366" alt="Screenshot 2024-06-03 at 08 03 48" src="https://github.com/khoj-ai/khoj/assets/65192171/3b0b770a-59b0-472e-bae2-a9b9b66179a0">